requirements.txt

```diff
@@ -14,4 +14,4 @@ PyYAML ==6.0.*
 pyOpenSSL==24.2.1
 kombu==5.4.1
 pymongo==4.8.0
-github_runner_manager @ git+https://github.com/canonical/github-runner-manager.git@771530633cd8b3cb18cb95399c234c82118b77e2
+github_runner_manager @ git+https://github.com/canonical/github-runner-manager.git@fix/wait_runner_check
```

Comment:
I think we should use the main branch commit, when that PR is merged right?

Reply:
Yup! Waiting for test pass!